docs/user_guide/usage_patterns/data_transfer/scheduled_transfers/create_timer.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+import globus_sdk
+from globus_sdk.experimental.globus_app import UserApp
+
+# Tutorial Client ID - <replace this with your own client>
+NATIVE_CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
+USER_APP = UserApp("manage-timers-example", client_id=NATIVE_CLIENT_ID)
+
+# Globus Tutorial Collection 1
+# https://app.globus.org/file-manager/collections/6c54cade-bde5-45c1-bdea-f4bd71dba2cc
+SRC_COLLECTION = "6c54cade-bde5-45c1-bdea-f4bd71dba2cc"
```

Comment:
Is this UUID now guaranteed? I was under the impression that collection might be recreated from time-to-time.

Reply:
I think there was a brief period when we transitioned from the original tutorial endpoints (go#ep1 and go#ep2) to the new tutorial collections during which we didn't have that guarantee. But a lot of us were not happy with that situation because it would mean that we can't rely on these IDs -- it's a problem not only here but also for docs.globus.org  -- so I'm pretty sure we "fixed" that instability.